AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -258,7 +258,7 @@ internal class CardViewController: FormViewController {
         let additionalItem = FormTextInputItem(style: formStyle.textField)
         additionalItem.title = localizedString(.cardTaxNumberLabelShort, localizationParameters)
         additionalItem.placeholder = localizedString(.cardTaxNumberPlaceholder, localizationParameters)
-        additionalItem.validator = LengthValidator(minimumLength: 6, maximumLength: 10)
+        additionalItem.validator = LengthValidator(exactLength: 10) || DateValidator(format: DateValidator.Format.kcpFormat)
```

Comment:
this is nice. can we write somewhere that it is 10 because that's the length of the tax number in Korea? :)

Reply:
Oh, good one!